### ISSUE TITLE
BsCarousel indicators are missing `aria-label` attribute.

### ISSUE THREAD

**Issue by SanderKnauff** [ember-bootstrap/ember-bootstrap#2079]
According to the bootstrap docs, the `aria-label` attribute should be present on the slide indicators.

See: https://getbootstrap.com/docs/5.3/components/carousel/#indicators

This only seems to apply to Bootstrap 5, as the Bootstrap 4 docs make no mention of the label.
https://getbootstrap.com/docs/4.6/components/carousel/
